src/appProvider.js

```diff
@@ -22,13 +22,16 @@ class AppProvider extends React.Component {
     super(props);
     this.state = { locale: defaultidx };
     this.langHandler = this.langHandler.bind(this);
```

Comment:
if we make `langHandler` a fat arrow function, we shouldn't need this in the constructor